payloads/v8/_interactions/_applicationCommands/chatInput.ts

```diff
@@ -20,6 +20,7 @@ interface APIApplicationCommandOptionBase {
 	description: string;
 	default?: boolean;
 	required?: boolean;
+	autocomplete?: never;
```

Comment:
Why are you defining it here as optionally never o.o

Reply:
It's weird, but this is what was I was told to do in the TS discord in order to get typechecking to work properly for the `autocomplete` field, while also preserving the union. I'm pretty sure this is the only way this can work.